stock_scanner_inventory/data/Inventory/scanner_scenario_step_inventory_product_code.py

```diff
@@ -0,0 +1,41 @@
+# flake8: noqa
+# Use <m> or <message> to retrieve the data transmitted by the scanner.
+# Use <t> or <terminal> to retrieve the running terminal browse record.
+# Put the returned action code in <act>, as a single character.
+# Put the returned result or message in <res>, as a list of strings.
+# Put the returned value in <val>, as an integer
+
+from datetime import datetime
+
+# No current inventory, create a new one
+if not terminal.tmp_val1:
+    stock_inventory_obj = env['stock.inventory']
+    stock_inventory_id = stock_inventory_obj.create({
+        'name': '%s : %s' % (terminal.code, datetime.today().strftime('%Y-%m-%d %H:%M:%S')),
```

Comment:
please use `fields.Datetime.now()`

Also just FYI `datetime.today()` is not a thing - I believe you were going for `datetime.now()`